cirq-core/cirq/ops/raw_types.py

```diff
@@ -580,6 +580,19 @@ def controlled_by(
         return ops.controlled_operation.ControlledOperation(control_qubits, self, control_values)
 
     def with_probability(self, probability: 'cirq.TParamVal') -> 'cirq.Operation':
+        """Creates a probabalistic channel with this operation.
+
+        Args:
+            probability: floating value between 0 and 1, giving the probability
```

Comment:
nit: floating value -> floating point value

Reply:
Done, also for `Gate.with_probability`.